addons/account/models/chart_template.py

```diff
@@ -244,23 +244,9 @@ def tax_template_changed(tax, template):
             )
 
         obsolete_xmlid = set()
+        custom_field = {'account.fiscal.position.tax_ids'}
```

Comment:
- we should be moved near its usage position
- we should add a comment explaining that we don't want to change our customers mappings